PlayCover/ViewModel/NetworkVM.swift

```diff
@@ -7,6 +7,7 @@
 
 import SystemConfiguration
 import Foundation
+import Semaphore
```

Comment:
Wasn't `import Semaphore` added before? I'm pretty much sure DispatchSemaphore can handle it without explicitly importing Semaphore

Reply:
Actually that is a mistaken import there, DispatchSemaphore is still in authority to check the NetworkVM not AsyncSemaphore. I'll remove.